### PR TITLE
[AMQ-9596] Add priority and disableMessageTimestamp to the producer command

### PR DESCRIPTION
**Pull request by kenliao94**
Jira tickets: https://issues.apache.org/jira/projects/AMQ/issues/AMQ-9596?filter=allissues

### CODE REVIEW

**Comment by pa-deasy** [comment]
Change LGTM. Thanks Ken.

File: activemq-client/src/main/java/org/apache/activemq/util/ProducerThread.java [hunk 1]
```diff
@@ -40,6 +40,8 @@ public class ProducerThread extends Thread {
     long msgTTL = 0L;
     String msgGroupID=null;
     int transactionBatchSize;
+    int priority = 4;
```
[the remark this answers is not in the document]
Comment (kenliao94) [reply activemq-client/src/main/java/org/apache/activemq/util/ProducerThread.java]: Good callout. Done.

File: activemq-console/src/main/java/org/apache/activemq/console/command/ProducerCommand.java [hunk 2]
```diff
@@ -45,6 +45,8 @@ public class ProducerCommand extends AbstractCommand {
     String msgGroupID=null;
     int transactionBatchSize;
     private int parallelThreads = 1;
+    int priority = 4; // Default priority
```
[the remark this answers is not in the document]
Comment (kenliao94) [reply activemq-console/src/main/java/org/apache/activemq/console/command/ProducerCommand.java]: Good callout. Done.

**Review by mattrpav — APPROVED** [review]
LGTM

**Review by jbonofre — APPROVED** [review]
Awesome work ! Thanks @kenliao94 !

**Pull request merged**